src/PIL/JpegImagePlugin.py

```diff
@@ -485,7 +485,7 @@ def _fixup_dict(src_dict):
 def _getexif(self):
     if "exif" not in self.info:
         return None
-    return dict(self.getexif())
+    return self.getexif()._get_merged_dict()
```

Comment:
`def _fixup_dict(src_dict):` defined above seems unused used

Reply:
Ok, I've suggested that change in a separate PR, #4957